### PR TITLE
TW-2018 Changed power levels in group chats

### PR DESCRIPTION
**Pull request by KhaledNjim**
## Ticket
#2018 

## Impact description
only the admin can :
-Change group chat avatar
-Change name of the group chat
-Change group chat description
-Add members
-Remove members

## Resolved
**Attach screenshots or videos demonstrating the changes**

https://github.com/user-attachments/assets/f6145003-4935-4a77-9b8d-021a13a81245

### CODE REVIEW

File: lib/pages/new_group/new_group_chat_info.dart [hunk 1]
```diff
@@ -106,6 +106,8 @@ class NewGroupChatInfoController extends State<NewGroupChatInfo>
         urlAvatar: urlAvatar,
         powerLevelContentOverride: {
           'events': powerLevelManager.getDefaultPowerLevelEventForMember(),
+          'invite': powerLevelManager.getAdminPowerLevel(),
+          'kick': powerLevelManager.getAdminPowerLevel(),
```
Comment (hoangdat) [inline lib/pages/new_group/new_group_chat_info.dart:110]: is it right spec? can you share docs. Thank you

File: lib/utils/power_level_manager.dart [hunk 2]
```diff
@@ -14,13 +14,15 @@ class PowerLevelManager {
...
+  int getAdminPowerLevel() => DefaultPowerLevelMember.admin.powerLevel;
+
   Map<String, dynamic> getDefaultPowerLevelEventForMember() {
     return {
       EventTypes.RoomPinnedEvents: getUserPowerLevel(),
-      EventTypes.RoomName: getUserPowerLevel(),
-      EventTypes.RoomAvatar: getUserPowerLevel(),
+      EventTypes.RoomName: getAdminPowerLevel(),
+      EventTypes.RoomAvatar: getAdminPowerLevel(),
       EventTypes.RoomMember: getUserPowerLevel(),
```
Comment (hoangdat) [inline lib/utils/power_level_manager.dart:24]: What is `room Member` event type? I think it relates to `invitation`, `kick`, `ban`? Is it?

File: lib/pages/new_group/new_group_chat_info.dart (hunk 1 above)
Comment (KhaledNjim) [reply lib/pages/new_group/new_group_chat_info.dart:110]: https://spec.matrix.org/v1.6/client-server-api/#creation
image
https://spec.matrix.org/v1.6/client-server-api/#mroompower_levels
image

File: lib/utils/power_level_manager.dart (hunk 2 above)
Comment (KhaledNjim) [reply lib/utils/power_level_manager.dart:24]: This event contains membership changes

**Review by hoangdat — APPROVED** [review]

**Review by nqhhdev — APPROVED** [review]
LGTM

**Pull request merged**